cmd/erasure.go

```diff
@@ -373,23 +375,22 @@ func (er erasureObjects) nsScanner(ctx context.Context, buckets []BucketInfo, wa
 
 	// Put all buckets into channel.
 	bucketCh := make(chan BucketInfo, len(buckets))
-	// Add new buckets first
```

Comment:
You should keep this functionality. Otherwise new buckets will take quite a while to show up in usage.

It will only be for one cycle anyway.